### ISSUE TITLE
[Feature Request] Format the phone numbers

### ISSUE THREAD

**Issue by K4LCIFER** [SimpleMobileTools/Simple-Contacts#1012]
Currently, the phone numbers are just unformated strings of numbers, for example `5555555555`. They should, instead, look like `(555) 555-5555`.

**Comment by inson1** [comment]
Another format is +555 555 555 555.

**Comment by JohnVeness** [comment]
This is going to vary from country to country.

**Comment by inson1** [comment]
@JohnVeness Yes, that would be default and then you could change it in settings imho.

**Comment by JohnVeness** [comment]
With any luck there's already a library for this kind of thing.

**Comment by andreyjc** [comment]
Please make the app at least separate the country code (e.g. +1) from the area code, etc.
